ci/dependency-update/Jenkinsfile

```diff
@@ -90,6 +90,11 @@ pipeline {
 		// Run at least once per week, in case of snapshot updates.
 		cron '@weekly'
 	}
+	parameters {
+		choice(name: 'UPGRADE_JOB', choices: ['all'] + configurations.keySet() as String[], defaultValue: 'all', description: 'Select which upgrade jobs to run. `All` will include any upgrade job that does not require building Hibernate ORM from sources.')
+		string(name: 'ORM_REPOSITORY', defaultValue: 'git@github.com:hibernate/hibernate-orm.git', description: 'Git URL to Hibernate ORM repository')
+		string(name: 'ORM_BRANCH', defaultValue: 'main', description: 'Hibernate ORM branch to build from.')
```

Comment:
I'm not sure default values make sense here... We'll always specify these parameters explicitly whenever they are useful, won't we? And I'm not sure building ORM main is a good default anyway...

If you need default values just so things work well in scheduled builds, maybe an empty string would be better, at least for the branch? That way we'd get an error if someone forgets to set it, at least.